easybuild/easyconfigs/p/Python/Python-2.7.3-intel-2015a.eb

```diff
@@ -0,0 +1,85 @@
+name = 'Python'
+version = '2.7.3'
+
+homepage = 'http://python.org/'
+description = "Python is a programming language that lets you work more quickly and integrate your systems more effectively."
+
+toolchain = {'name': 'intel', 'version': '2015a'}
+toolchainopts = {'pic': True, 'opt': True, 'optarch': True}
+
+numpyversion = '1.6.1'
+scipyversion = '0.10.1'
+
+source_urls = ['http://www.python.org/ftp/%(namelower)s/%(version)s/']
+sources = [SOURCE_TGZ]
+
+patches = ['python_libffi_int128_icc.patch']
+
+# python needs bzip2 to build the bz2 package
+dependencies = [
+    ('bzip2', '1.0.6'),
+    ('zlib', '1.2.7'),
```

Comment:
Don't agree here. We stick the zlib to the toolchain. 2 different zlibs for one toolchain is trouble.